src/components/organize/tasks/TaskList/TaskListItem.tsx

```diff
@@ -0,0 +1,94 @@
+import { createStyles } from '@material-ui/core/styles';
+import { FormattedMessage } from 'react-intl';
+import NextLink from 'next/link';
+import { Theme } from '@material-ui/core/styles';
+import { Card, Link, ListItem, makeStyles, Typography } from '@material-ui/core';
+
+import ZetkinRelativeTime from '../../../ZetkinRelativeTime';
+import { ZetkinTask } from '../../../../types/zetkin';
+
+import TaskStatusChip from '../TaskStatusChip';
+import getTaskStatus, { TASK_STATUS } from '../../../../utils/getTaskStatus';
+
+interface TaskListItemProps {
+    task: ZetkinTask;
+    hrefBase: string;
+}
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+    MuiCard: {
+        '&:hover': {
+            color: theme.palette.primary.main,
+        },
+        padding: '1rem',
+        width: '100%',
+    },
+}));
+
+const TaskListItem = ({ task, hrefBase }: TaskListItemProps): JSX.Element => {
+    const classes = useStyles();
+    const { id, title, published, deadline, expires } = task;
+    const taskStatus = getTaskStatus(task);
+
+    return (
+        <NextLink href={ hrefBase + `/tasks/${id}` } passHref>
+            <Link color="inherit" underline="none" variant="subtitle2">
+                <ListItem>
+                    <Card className={ classes.MuiCard }>
+                        <TaskStatusChip status={ taskStatus }/>
+
+                        <Typography component="h3" variant="h5">
+                            { title }
+                        </Typography>
+
+                        <Typography color="textPrimary">
+                            { /* Scheduled */ }
+                            { taskStatus === TASK_STATUS.SCHEDULED && published && (
+                                <>
+                                    <FormattedMessage id="misc.tasks.taskListItem.relativeTimes.scheduled" />
+                                    { ' ' }
+                                    <ZetkinRelativeTime datetime={ published } />
```

Comment:
These should be refactored to pass the relative time as a parameter to the `<FormattedMessage>` instead of rendering the relative time after the message. In some languages, this might not be the grammatically correct way to represent it. I honestly don't know if this is the case for any (supported) languages, but maybe German where verbs are after prepositions? Anyway, the option should remain available to the translator.

You can do this:

```tsx
<FormattedMessage
  id="misc.tasks.taskListItem.relativeTimes.scheduled"
  values={{ time: <ZetkinRelativeTime datetime={ published }/> }}
/>
```

And then in the YAML you can do this:

```yaml
taskListItem:
    relativeTimes:
        scheduled: Will be published { time }
```

This would allow a translator to another language to do this:

```yaml
taskListItem:
    relativeTimes:
        scheduled: wird { time } veröffentlicht
```

Reply:
Done!